duplicacy/duplicacy_main.go

```diff
@@ -687,14 +689,20 @@ func restoreRepository(context *cli.Context) {
             pattern = pattern[1:]
         }
 
-        if pattern[0] != '+' && pattern[0] != '-' {
+        if duplicacy.IsUnspecifiedFilter(pattern) {
             pattern = "+" + pattern
         }
 
-        if pattern == "+" || pattern == "-" {
+        if duplicacy.IsEmptyFilter(pattern) {
             continue
         }
 
+        valid, err := duplicacy.IsValidRegex(pattern)
```

Comment:
Same question here about how this code appears to be requiring all patterns to be regexes.

Reply:
oops, yes, has been fixed in latest PR.